src/sonic-frr/patch/0006-frr-remove-frr-log-outchannel-to-var-log-frr.log.patch

```diff
@@ -1,14 +1,8 @@
-From 31464e3fd0f677bc15286e99d4a30e2e33480c53 Mon Sep 17 00:00:00 2001
-From: Ying Xie <ying.xie@microsoft.com>
-Date: Wed, 25 Nov 2020 22:28:43 +0000
-Subject: [PATCH] [frr] remove frr log outchannel to /var/log/frr.log
+From 0417036a3aece862c95111d4646ca7508a028165 Mon Sep 17 00:00:00 2001
+From: Syed Hasan Raza Naqvi <syed.naqvi@broadcom.com>
+Date: Thu, 21 Apr 2022 17:25:02 -0700
+Subject: [PATCH] [PATCH 6/8] [frr] remove frr log outchannel to /var/log/frr.log
 
-SONiC runs frr inside a docker and the logs are sent to base image
```

Comment:
can you please not remove the original comments for the patch?

Reply:
Done.